src/dmd/link.d

```diff
@@ -759,21 +777,45 @@ public int runLINK()
             perror("unable to create pipe to linker");
             return -1;
         }
-        childpid = fork();
-        if (childpid == 0)
+
+        version (CRuntime_Glibc)
```

Comment:
This could also be done on Musl, couldn't it ?

Reply:
For future reference: [struct on musl](http://git.musl-libc.org/cgit/musl/tree/include/spawn.h#n36)

Reply:
Um, I checked `spawn.d` and it could be done on many, many systems. At some point I'm just copying that file into dmd...